plugin/evm/validators_ftso_test.go

```diff
@@ -0,0 +1,40 @@
+package evm
+
+import (
+	"math/rand"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWithRootDEgree(t *testing.T) {
+	t.Run("Signature", func(t *testing.T) {
+		var (
+			ftsoOption        interface{}
+			signatureExpected = "evm.FTSOOption"
+		)
+
+		for i := 0; i < 10; i++ {
+			degree := rand.Intn(1000)
+			ftsoOption = WithRootDegree(uint(degree))
+			signature := reflect.TypeOf(ftsoOption).String()
+
+			assert.Equal(t, signature, signatureExpected)
+		}
+	})
+
+	t.Run("CfgMutation", func(t *testing.T) {
+		testCfg := FTSOConfig{
+			RootDegree: uint(9),
+		}
+
+		for i := 0; i < 25; i++ {
+			degree := rand.Intn(1000)
+			ftsoOption := WithRootDegree(uint(degree))
+			ftsoOption(&testCfg)
+
+			assert.Equal(t, testCfg.RootDegree, uint(degree))
+		}
+	})
```

Comment:
Same suggestion here. Test should be simplified.

Reply:
Please note that I removed the initial version here also:
https://github.com/flare-foundation/coreth/pull/21/commits/5c646cdde6cb04d81b0d03b20e2bcd58846aeb6c#diff-5d4bfd4a98a9e3484f1bb6efb6fee69a6b752832d96c0ed354bec7be522ce905
Thanks!